<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDEventConditionDialog_CycleTimer</class>
 <widget class="QDialog" name="CMDEventConditionDialog_CycleTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>周期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cycleEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cycleEditSecond">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cycleEditMill">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>毫秒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
